### PR TITLE
Sdf world scene

### PR DESCRIPTION
**Pull request by gustavoneves12**
create sdf scene in the C/C++ side
create and manage the ports used to communicate with other tasks.

### CODE REVIEW

File: vizkit3d_worldTypes.hpp [hunk 1]
```diff
@@ -8,7 +8,17 @@
  * which case you do not need this file
  */
 
+#include <iostream>
```
Comment (doudou) [inline vizkit3d_worldTypes.hpp]: This is now unused, get rid of it

Comment (doudou) [inline vizkit3d_worldTypes.hpp]: (I mean, the whole ModelState type)

File: vizkit3d_world.orogen [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
 
 using_library "vizkit3d_world"
 
-
 task_context "Task" do
     needs_configuration
     
+    property "model_paths", "/std/vector</std/string>"
     property "world_file_path", "/std/string"
+    property "show_gui", "bool"
```
Comment (doudou) [inline vizkit3d_world.orogen]: Make this false by default IMO

Comment (doudou) [inline vizkit3d_world.orogen]: And document each property / port. Just add a comment on top of it, e.g.

```
# Paths to resolve the model:// URIs in the world file
property "model_paths", ...
```

**Comment by gustavoneves12** [comment]
@doudou PING

**Pull request merged**